Fordítások/CrazyAuctions/Messages.yml

```diff
@@ -1,39 +1,39 @@
-  Not-A-Number: '&c%Arg% is not a number.'
-  Got-Item-Back: '&7Your item has been returned.'
-  Cancelled-Item: '&7You have cancelled an item on the auction list, return your items with /ah expired.'
-  Item-Has-Expired: '&7An item you have in the Crazy Auctions has just expired.'
-  Admin-Force-Cancelled: '&7You have just force cancelled a sale.'
-  Admin-Force-Cancelled-To-Player: '&cOne of your items was just force cancelled by an Admin.'
-  Item-Doesnt-Exist: '&cThat item isnt in the crazy auctions any more.'
-  Max-Items: '&cYou cant list any more items to the Crazy Auctions.'
-  Item-BlackListed: '&cThat item is not allowed to be sold here.'
-  Item-Damaged: '&cThat item is damaged and is not allowed to be sold here.'
-  Sold-Msg: '&7Thank you for buying this item.'
-  Bid-More-Money: '&cYour bid is to low, please bid more.'
-  Not-A-Currency: '&cThat is not a currency. Please use Money or Tokens.' #Remove Tokens if you do not have TokenManager.
-  Sell-Price-To-Low: '&cYour sell price is to low the minimum is &a$10&c.'
-  Sell-Price-To-High: '&cYour sell price is to high the maximum is &a$1000000&c.'
-  Bid-Price-To-Low: '&cYour starting bid price is to low the minimum is &a$100&c.'
-  Bid-Price-To-High: '&cYour starting bid price is to high the maximum is &a$1000000&c.'
-  Bought-Item: '&7You have just bought a item for &a$%Price%&7.'
-  Win-Bidding: '&7You have just won a bid for &a$%Price%&7. Do /Ah Collect to collect your winnings.'
-  Player-Bought-Item: '&7%Player% has bought your item for &a$%Price%.'
-  Someone-Won-Players-Bid: '&7%Player% has won your item you from a bid for &a$%Price%.'
-  Added-Item-To-Auction: '&7You have just added a item to the crazy auctions for &a$%Price%&7.'
-  Bid-Msg: '&7You have just bid &a$%Bid% &7on that item.'
+  Players-Only: '&cCsak játékosok használhatják ezt a parancsot.'
+  Reload: '&7Újratöltötted az Őrült Aukciók fájlát.'
+  Need-More-Money: '&cSzükséged van több &a$%Money_Needed%&c-re.'
+  Inventory-Full: '&cTele van a leltárad. Szabadíts fel helyet a megvásárlásához.'
+  No-Permission: '&cNincs engedélyed a parancs használatához!'
+  Not-Online: '&cEz a játékos jelenleg nincs online.'
+  Doesnt-Have-Item-In-Hand: '&cA kezedben kell lennie az eladandó tárgynak.'
+  Not-A-Number: '&c%Arg% nem egy szám.'
+  Got-Item-Back: '&7A tárgyad visszatért hozzád.' 
+  Cancelled-Item: '&7Megszűntetted egy tárgyad vehetőségét, az aukcióról levett tárgyad a /ah expired paranccsal tudod megnézni.'
+  Item-Has-Expired: '&7Lejárt az egyik eladni kívánt tárgyad.'
+  Admin-Force-Cancelled: '&7Kivettél egy tárgyad az aukcióról.'
+  Admin-Force-Cancelled-To-Player: '&cAz egyik eladni kívánt tárgyadat az egyik admin kivette.'
+  Item-Doesnt-Exist: '&cEz a tárgy nincs többé az aukcióban.'
+  Max-Items: '&cJelenleg nem tudsz több tárgyat az Őrült Akuciókban.'
+  Item-BlackListed: '&cEz a tárgyat nem adhatod el.'
+  Item-Damaged: '&cEz a tárgy megsérült, vagy nem engedélyezett az eladása.'
```

Comment:
&cThat item is not allowed to be sold here. -> &cEz a tárgy sérült és ez nem engedélyezett eladásra.